### PR TITLE
feat: Tabbing Order E2E Tests - Order History

### PR DESCRIPTION
**Pull request by Zeyber**
Closes: #4212

### CODE REVIEW

File: projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order/order-history.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  cy.visit('/my-account/orders');
+  cy.get('cx-order-history a')
+    .first()
+    .focus();
+  checkAllElements(config);
+}
+
+export function orderHistoryWithOrdersTabbingOrder() {
+  doPlaceOrder().then((orderData: any) => {
+    const config: TabElement[] = [
```
Comment (kacperknapik) [inline projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order/order-history.ts]: please create 2 entries in our config.
- orderHistoryNoOrders
- orderHistoryOrders

File: projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-order.e2e-spec.ts [hunk 2]
```diff
@@ -89,6 +90,18 @@ context('Tabbing order - tests do require user to be logged in', () => {
...
 
+  describe('Order History', () => {
+    it('should allow to navigate with tab key (no orders)', () => {
+      orderHistoryNoOrdersTabbingOrder(config.orderHistoryNoOrders);
+    });
+
+    it('should allow to navigate with tab key (with orders)', () => {
+      cy.window().then(win => win.sessionStorage.clear());
+      cy.requireLoggedIn();
+      orderHistoryWithOrdersTabbingOrder();
```
Comment (kacperknapik) [inline projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-order.e2e-spec.ts]: you can use the second config here (mentioned above)

File: projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-order.e2e-spec.ts [hunk 3]
```diff
@@ -89,6 +90,18 @@ context('Tabbing order - tests do require user to be logged in', () => {
     cy.saveLocalStorage();
   });
 
+  describe('Order History', () => {
+    it('should allow to navigate with tab key (no orders)', () => {
+      orderHistoryNoOrdersTabbingOrder(config.orderHistoryNoOrders);
+    });
+
+    it('should allow to navigate with tab key (with orders)', () => {
+      cy.window().then(win => win.sessionStorage.clear());
```
Comment (kacperknapik) [inline projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-order.e2e-spec.ts]: can u explain what's going on here?

File: projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order/order-history.ts (hunk 1 above)
Comment (Zeyber) [reply projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order/order-history.ts]: Config for orderHistoryOrders needs to be dynamically generated based on the result from `doPlaceOrder()` and can't just be placed in our config.

File: projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-order.e2e-spec.ts (hunk 3 above)
Comment (Zeyber) [reply projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-order.e2e-spec.ts]: I'm not sure why but it works when you do this.

File: projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-order.e2e-spec.ts (hunk 2 above)
Comment (Zeyber) [reply projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-order.e2e-spec.ts]: Can't for the reason mentioned above

File: projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-order.e2e-spec.ts (hunk 3 above)
Comment (Zeyber) [reply projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-order.e2e-spec.ts]: There is probably a better solution which could be introduced as an optimization.

**Pull request merged**